BridgeSDK/BridgeAPI/SBBActivityManager.m

```diff
@@ -112,18 +139,62 @@ - (void)mapSubObjectsInTaskList:(NSMutableArray *)taskList
     }
 }
 
-- (NSURLSessionDataTask *)getScheduledActivitiesForDaysAhead:(NSInteger)daysAhead daysBehind:(NSInteger)daysBehind cachingPolicy:(SBBCachingPolicy)policy withCompletion:(SBBActivityManagerGetCompletionBlock)completion
+// make sure to obtain activityAccessLock before calling these next three methods, and keep it until after all three
+// have been called! also check gSBBUseCache before calling.
+- (SBBResourceList *)cachedTasksAndCacheManager:(id<SBBCacheManagerProtocol> *)cacheManager
```

Comment:
Could you change `cacheManager` to `outCacheManager` to differentiate it from `theCacheManager`? If I were modifying this, I might not notice the difference.